packages/jest-jasmine2/src/index.js

```diff
@@ -92,8 +92,9 @@ function jasmine2(
     env.specFilter = spec => testNameRegex.test(spec.getFullName());
   }
 
+  runtime.requireModule(require.resolve('p-map'));
```

Comment:
i thought that would help, but it didn't :) 
i ended up removing p-map completely and that helped. I'll remove this line